.github/workflows/automate-sample-apps.yml

```diff
@@ -0,0 +1,18 @@
+name: Trigger a repository dispatch event to sample apps repo to run apps
+
+on:
+  push:
+    branches: [ master, v1, v2-beta ]
+  pull_request:
+    branches: [ master, v1, v2-beta ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    
+    steps:
+    - uses: actions/checkout@v1
+    - name: Send a dispatch event
+      run: |
+          curl -X POST https://api.github.com/repos/MicrosoftOryx/githubactions-samples/dispatches -H 'Accept: application/vnd.github.everest-preview+json' -H 'Authorization: token ${{secrets.REPO_ACCESS_TOKEN}}' --data '{"event_type": "build_application"}'
```

Comment:
To make it easy for anyone of us (collaborators) to kick off a build on-demand, I think having this line in the Readme of the samples repo would be convenient.

Reply:
That makes sense. I'll add one there either. Thanks!